src/components/TaskCard.vue

```diff
@@ -1,57 +1,164 @@
+                            {{ newPriority }}
+
+                            <div v-if="task.editing">
+                                <img v-if="!priorChooser" src="@/assets/arrowDown.png" class="w-3 pl-1">
+                                <img v-else src="@/assets/arrowDownB.png" class="w-3 pl-1">
+                            </div>
+                        </button>
+
+                        <!-- priority dropdown -->
+                        <div v-if="priorChooser"
+                            class="absolute top-full left-0 mt-1 w-full bg-white rounded-2xl border-2 z-10">
+                            <ul class="flex flex-col text-sm text-black">
+                                <li v-for="(priority, index) in priorities"
+                                    @click="newPriority = priority.level; priorChooser = false"
+                                    class="px-2 hover:bg-gray-100" :class="[index === 0 ? 'rounded-t-2xl pt-2' : '',
+                                    index === 2 ? 'rounded-b-2xl pb-2' : '']">{{ priority.level }}
+                                </li>
+                            </ul>
+                        </div>
                     </div>
-                </button>
-            </div>
 
-            <!-- task info -->
-            <div :class="isMobile ? 'order-2 pl-4' : 'order-1 w-3/4'">
-                <h3 class="font-semibold text-black text-2xl">{{ task.name }}</h3>
-                <p class="text-gray-500 w-xs">{{ isMobile ? task.date : task.desc }}</p>
+                    <!-- task priority on phone -> while being actively edited -->
+                    <div v-if="isMobile && task.editing" class="flex flex-row order-3 space-x-2 ">
+                        <button v-for="priority in priorities" @click="newPriority = priority.level"
+                            class="rounded-full order-3 h-3 w-3 focus:outline-2"
+                            :class="getBgColor(priority.level)"></button>
+                    </div>
+                </div>
+
+                <div class="flex flex-row">
+                    <div class="flex flex-col">
+                        <div v-if="isMobile" class="flex flex-row space-x-2 content-center">
+                            <font-awesome-icon v-if="task.editing" :icon="['fas', 'calendar-days']" />
+                            <p class="w-xs" :class="task.editing ? '' : 'text-gray-500'">{{ task.date }}</p>
+                        </div>
+                        <!-- resize on bigger text? -->
+                        <textarea v-if="!isMobile || task.editing" v-model="newDesc"
+                            placeholder="Here goes the description." class=" w-90 outline-0 resize-none "
+                            :class="task.editing ? 'h-20' : 'h-7'"></textarea>
+                    </div>
+
+                    <!-- done button on desktop -->
+                    <button v-if="!isMobile && !task.editing" @click.stop @click="$emit('done-clicked', task.id)"
```

Comment:
you can merge the two click handlers, like this: @click.stop="$emit('done-clicked', task.id)"